tiered-storage/file-system/src/main/java/org/apache/bookkeeper/mledger/offload/filesystem/impl/FileSystemManagedLedgerOffloader.java

```diff
@@ -62,12 +65,14 @@
     private final FileSystem fileSystem;
     private OrderedScheduler scheduler;
     private static final long ENTRIES_PER_READ = 100;
+    private static final int PREFETCH_ROUNDS = 100;
```

Comment:
One per reading is ok, the PREFETCH_ROUNDS unnecessary to big.

Reply:
ok